5주차/mission/.env

```
@@ -0,0 +1,21 @@
+VITE_SERVER_API_URL=http://localhost:8000
+
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: <https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema>
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: <https://pris.ly/d/connection-strings>
+
+DATABASE_URL="file:./dev.db"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)